src/index.html

```diff
@@ -5,4 +5,151 @@
 order: 1
 ---
 
-<p>Add the HTML for your homepage here.</p>
+<iframe src="https://www.youtube.com/embed/z_jefhcVRAk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<div class="about">
+    <div class="img">
+        <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fabout.png?v=1605143407347" alt="Dancing Girl">
+    </div>
```

Comment:
You likely don't need this `div` yet -- again, you may end up needing extra markup, but you don't want to start with it.